draft-ietf-cbor-serialization.md

```diff
@@ -539,17 +549,14 @@ While this is technically possible in CBOR, it comes with significant drawbacks:
 - Values cannot be translated directly to JSON, which does not support NaNs of any kind.
 
 
-## Clarification of {{-cbor}}
-
-This is a clarifying restatement of how NaNs are to be treated according to {{-cbor}}.
+## Clarification of RFC 8949
 
-NaNs represented in floating-point values of different lengths are considered equivalent in the basic generic data model if:
+This is a clarifying restatement of how NaNs are treated according to {{-cbor}}.
 
- - Their sign bits are identical, and
- - Their significands are identical after both significands are zero-extended on the right to 64 bits
+CBOR floating point values are based on 64-bit floating point values.
+NaNs represented in floating-point values of different lengths are considered equivalent by CBOR in the basic generic data model if a) their sign bits are identical, and b) their significands are identical after both significands are zero-extended on the right to 64 bits
 
-This equivalence is established for the entire CBOR basic generic data model.
-A NaN encoded as half-, single-, or double-precision is equivalent whenever it satisfies the rules above.
+A NaN encoded as half-, single-, or double-precision is equivalent in CBOR whenever it satisfies the rules above.
```

Comment:
The statement about equivalence in the data model is critical and can't be removed.